internal/tekton-bundle/tekton-bundle.go

```diff
@@ -38,138 +36,107 @@ type Bundle struct {
 	ConfigMaps      []v1.ConfigMap
 }
 
-func ReadTasksBundle(isOpenshift bool) (*Bundle, error) {
-	var files [][]byte
-	path := getTasksBundlePath(isOpenshift)
-	data, err := os.ReadFile(path)
-	if err != nil {
-		return nil, err
-	}
-	files = append(files, data)
-
-	tektonObjs, err := decodeObjectsFromFiles(files)
-	if err != nil {
-		return nil, err
-	}
-
-	return tektonObjs, nil
-}
-
-func ReadPipelineBundle() (*Bundle, error) {
-	path := getPipelineBundlePath()
-	files, err := readFolder(path)
-	if err != nil {
-		return nil, err
-	}
-
-	tektonObjs, err := decodeObjectsFromFiles(files)
-	if err != nil {
-		return nil, err
-	}
-
-	return tektonObjs, nil
-}
-
-func getPipelineBundlePath() string {
-	return tektonPipelinesBundleDir
+func GetTektonPipelineBundlePaths() []string {
+	var paths []string
```

Comment:
Can you assemble this slice one time as a global? Usually I dislike globals, but for storing this kind of information it should be fine. It saves recreating the slice on every func call.

Reply:
Okay done.